cogs5e/models/character.py

```diff
@@ -209,6 +209,12 @@ async def from_bot_and_ids(cls, bot, owner_id: str, character_id: str):
         cls._cache[owner_id, character_id] = inst
         return inst
 
+    @classmethod
+    async def serialize_character_from_dict(cls, owner_id, character_dictionary):
+        char = Character.from_dict(character_dictionary)
+        cls._cache[owner_id, character_dictionary["upstream"]] = character_dictionary
```

Comment:
Based on the other uses of the cache in this file, it looks like the cache should be holding a `Character` instance, not the character dictionary. Maybe you meant `cls._cache[owner_id, character_dictionary["upstream"]] = char`?